src/utils/api/auth.ts

```diff
@@ -43,6 +47,15 @@ class AuthApi {
         await this._wait(500);
         return { status: 'success' };
     }
+
+    async editPassword({ password, confirmPassword }: EditPasswordData): Promise<{ status: 'success' | 'error'; data?: User; errorMessage?: string }> {
+        await this._wait(500);
+        if (confirmPassword && password) {
+            return { status: 'success', data: { name: 'User', phone: '+7 999 99 99 99', isOwner: false } };
```

Comment:
```suggestion
            return { status: 'success', data: { name: 'User', phone: '+7 999 999 99 99', isOwner: false } };
```

вы с бэками договрились про номера телефонов, можно ли с ними делать пробелы или скобочки?